lib/chef/cookbook/cookbook_version_loader.rb

```diff
@@ -231,12 +231,12 @@ def load_all_files
 
             Dir.glob(File.join(fs_entry, "**/*"), File::FNM_DOTMATCH).each do |file|
               next if File.directory?(file)
-              file = Chef::Util::PathHelper.cleanpath(file)
```

Comment:
Need a `.to_s` after each of these two changes so we don't change the type of the stuff in `all_files`.